commit 1f96f7e62bdfe17a489dee0dabfed64d61c5578e (refs/changes/62/4762/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-29T10:41:38-07:00 (1 year, 1 month ago)
    
    MANTA-3125 deliver libmanta via npm registry instead of git repo (npmignore file to limit cruft from published package)

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..3136e11
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,10 @@
+/.gitmodules
+/.dir-locals.el
+/Makefile
+/deps
+/docs
+/test
+/tools
+/tmp
+/npm-debug.log
+/libmanta-*.tgz
